.github/workflows/benchmark-command.yml

```diff
@@ -25,12 +26,15 @@ on:
           - high
           - good
           - medium
+          - dev
       benchmark:
         description: 'Benchmark to run'
         required: true
         options:
           - full
           - test
+          - small
+          - timeseries_test
```

Comment:
same here, these value options are a bit confusing

Reply:
I can make changes to keep a better naming convention